tsl/test/shared/sql/chunk_move_copy.sql

```diff
@@ -0,0 +1,78 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+CREATE TABLE hyper (time bigint NOT NULL, value integer);
```

Comment:
My apology for bringing a new comment. I just realised that this test will share the same database and run in parallel with other tests, thus it is good create a table with more unique name, e.g., the name of the test file, which is what you see in other test files.

Reply:
Also note that `include/shared_setup.sql` is executed before calling this file.

Reply:
> Also note that `include/shared_setup.sql` is executed before calling this file.

I believe you are already utilising the fact this shared script creates data nodes.

Reply:
done...